common/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/workspace/LaunchLiferay7WorkspaceAction.java

```diff
@@ -14,8 +35,25 @@ public LaunchLiferay7WorkspaceAction()
         super( "Liferay 7 Workspace" );
     }
 
+
     @Override
     public void run()
     {
+        LaunchWorkspaceHandler hanlder = new LaunchWorkspaceHandler();
+
+        Map<String, String> map = new HashMap<String, String>();
+
+        // TODO which workspace shall we point to ?
+        map.put( LaunchWorkspaceHandler.PARAM_WORKSPACE_LOCATION, "D:/test/forlaunch" );
```

Comment:
Can you put this value as anything?  Does it really matter?  Can you just use the string "workspace"?